containers/ecr-viewer/src/app/api/save-fhir-data/service.ts

```diff
@@ -308,13 +308,21 @@ const saveRR = async (
   // Loop through each condition/rule object in rr array
   for (const rrItem of metadata.rr) {
     const rr_conditions_uuid = randomUUID();
+
+    const rr_code = await trx
+      .selectFrom("condition_reference")
+      .select("code")
+      .where("condition_name", "=", rrItem.condition)
+      .executeTakeFirst();
```

Comment:
still needs to be sourced from the metadata data (and metadata parser updated to add it)